tests/postthrow.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+Check if throwing an exception in post hook after IO operation will finish gracefully
```

Comment:
So just to confirm that the i/o operation is critical to the bug being triggered? If `helloWorld()` didn't have that `scandir` call, the segfault wouldn't happen?

Reply:
Yes. I could not trigger it without some kind of IO operation.